ios/ElectrodeReactNativeBridge/ElectrodeBridgeHolder.m

```diff
@@ -21,6 +21,43 @@
 
 NS_ASSUME_NONNULL_BEGIN
 
+@interface ElectrodeQueuedEventListener: NSObject
+@property (nonatomic, strong) NSUUID *uuid;
+@property (copy) ElectrodeBridgeEventListener listener;
+- (instancetype) initWithUUID: (NSUUID *) uuid
+                     listener: (ElectrodeBridgeEventListener) listener;
+@end
+
+@implementation ElectrodeQueuedEventListener
+- (instancetype) initWithUUID: (NSUUID *) uuid
+                     listener: (ElectrodeBridgeEventListener) listener {
+    if (self = [super init]) {
+        _uuid = uuid;
+        _listener = listener;
+    }
+    
+    return self;
+}
+@end
+
+@interface ElectrodeQueuedRequestHandler: NSObject
+@property (nonatomic, strong) NSUUID *uuid;
+@property (copy) ElectrodeBridgeRequestCompletionHandler handler;
```

Comment:
Proposal: having a property with both atomic and copy seems a bit unusual, so I would propose writing the "atomic" here, too, to document that this was done on purpose.